.stylelintrc.yaml

```diff
@@ -0,0 +1,51 @@
+extends:
```

Comment:
Using a yaml config file as allows comments, so rules can easily be enabled or disabled.